### ISSUE TITLE
[BUG] status page挂了

### ISSUE THREAD

**Issue by zrt** [OI-wiki/OI-wiki#3200]
- [x] 我已经读过了 [F.A.Q.](https://oi-wiki.org/intro/faq/)，进行了搜索，但没有得到答案
- [ ] 我正在着手修复这个问题

## 我遇到了这样的问题（最好截图）
README首页有写：

> 我们在 status.oi-wiki.org 维护了一份镜像站列表，它们的内容和 oi-wiki.org 都是相同的。

然后[这个地址](status.oi-wiki.org)挂了。

## 我确认这个问题可以这样复现

与 #1947 不同，这次好像是因为uptime robot自定义域名功能变成收费了。

建议README上改为正确地址，或者status.oi-wiki.org跳转到uptime robot的正确地址。

另外我使用`crontab @daily git pull gh-pages分支`的方式镜像了一个oi-wiki，服务器在腾讯云香港，速度还不错，应该可以长期(>1年)镜像，可以收录一下。
地址是 https://oi-wiki.logits.org/ 。

**Comment by Ir1d** [comment]
是的， uptimebot 变收费了，最近一直没更新它，等我答辩完研究一下别的方案hh
谢谢 ruotian~

**Comment by CoelacanthusHex** [comment]
https://oi-wiki.org/intro/mirrors/
我们创建了一份镜像站列表，尽管目前没有状态监控

**Comment by Ir1d** [comment]
readme 还得改一下

**Comment by Enter-tainer** [comment]
> [oi-wiki.org/intro/mirrors](https://oi-wiki.org/intro/mirrors/) 我们创建了一份镜像站列表，尽管目前没有状态监控
> 
> > <img alt="" width="48" src="../../favicon.ico">**[镜像站列表 - OI Wiki](https://oi-wiki.org/intro/mirrors/)**OI Wiki 是一个编程竞赛知识整合站点，提供有趣又实用的编程竞赛知识以及其他有帮助的内容，帮助广大编程竞赛爱好者更快更深入地学习编程竞赛

https://status.oi-wiki.org/ 状态监控回来了
